projects/storefrontlib/cms-components/product/carousel/product-carousel/product-carousel.module.ts

```diff
@@ -31,7 +34,7 @@ import { ProductCarouselComponent } from './product-carousel.component';
       },
     }),
   ],
-  declarations: [ProductCarouselComponent],
-  exports: [ProductCarouselComponent],
+  declarations: [ProductCarouselComponent, ProductCarouselItemComponent],
+  exports: [ProductCarouselComponent, ProductCarouselItemComponent],
```

Comment:
Perhaps we create a module for `ProductCarouselItemComponent`.  What do you think?

Reply:
I'm wondering if that's unnecessary - the `ProductListModule` declares `ProductListItemComponent`, etc.
Would the benefit be for lazy loading?